.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install: 
+  pip install -r pip-requirements.txt
+
+# command to run tests
+script: 
+  nosetests -v --with-coverage --cover-package=presenters,plotting.figuremanager
```

Comment:
For the local tests that is good, but for PRs you can set limits in coveralls. You can specify that the build should fail if coverage goes down at all, or a certain percentage.